pkgs/development/python-modules/typecode/libmagic.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pkgs
```

```suggestion
, file
, zlib
```

Please inherit if a python package with the name exists.